src/package/nuspec/Microsoft.TestPlatform.nuspec

```diff
@@ -489,6 +489,17 @@
     <file src="net451\$Runtime$\CUITPlugins\Microsoft.VisualStudio.TestTools.UITest.Extension.UiaWidget.UIAHtmlElementUtilities.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\CUITPlugins\Microsoft.VisualStudio.TestTools.UITest.Extension.UiaWidget.UIAHtmlElementUtilities.dll" />
 
     <!-- External dependencies-->
-    <file src="$FakesPackageDir$\net451\Microsoft.VisualStudio.TestPlatform.Fakes.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestPlatform.Fakes.dll" />
+    <file src="$FakesPackageDir$\TestPlatform\Microsoft.VisualStudio.TestPlatform.Fakes.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestPlatform.Fakes.dll" />
+
+    <file src="$FakesPackageDir$\DataCollector\Microsoft.VisualStudio.Fakes.DataCollector.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Extensions\Microsoft.VisualStudio.Fakes.DataCollector.dll" />
+    <file src="$FakesPackageDir$\DataCollector\Microsoft.Cci.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Extensions\Microsoft.Cci.dll" />
+
+    <file src="$FakesPackageDir$\Instrumentation\x86\Microsoft.QualityTools.Testing.Fakes.Instrumentation.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Extensions\Fakes\x86\Microsoft.QualityTools.Testing.Fakes.Instrumentation.dll" />
+    <file src="$FakesPackageDir$\Instrumentation\x86\FakesInstrumentationProfiler_x86.config"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Extensions\Fakes\x86\FakesInstrumentationProfiler_x64.config" />
```

Comment:
The config linked does not seem right. This should be the x86 config...

Reply:
Fixed. Validated x64, x86 for netcoreapp3.1 and net472